src/ray/gcs/gcs_server/test/gcs_resource_manager_test.cc

```diff
@@ -38,7 +38,8 @@ TEST_F(GcsResourceManagerTest, TestBasic) {
   std::unordered_map<std::string, double> resource_map;
   resource_map[cpu_resource] = 10;
   ResourceSet resource_set(resource_map);
-  gcs_resource_manager_->UpdateResources(node_id, resource_set);
+  gcs_resource_manager_->UpdateTotalResources(node_id, resource_map);
+  gcs_resource_manager_->UpdateAvailableResources(node_id, resource_set);
```

Comment:
Is it necessary to invoke `UpdateAvailableResources`? I think the available resources are already updated by `UpdateTotalResources `.

Reply:
Yeah, good catch, thanks.